.github/workflows/solutions-dpg-qa.yaml

```
@@ -0,0 +1,156 @@
+  #      - name: Verify kubectl get nodes
+  #        run: |
+  #          kubectl get nodes
+
+  DIGIT-deployment:
+    #    if: ${{ github.event_name == 'push' }}
+    #    needs: Terraform_Infra_Creation
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)